README.md

```diff
@@ -43,7 +43,7 @@ Once you have all the dependencies set up, you can run the following:
 ```
 cargo run -p graph-node --release -- \
   --postgres-url postgresql://USERNAME[:PASSWORD]@localhost:5432/graph-node \
-  --ethereum-rpc [URL] \
+  --ethereum-rpc [NETWORK_NAME]:[URL] \
```

Comment:
I think it should be `NETWORK_NAME:[CAPABILITIES]:URL`, like it's described [here](https://github.com/graphprotocol/graph-node/blob/ff20590829230c9d3ed0751247ed39a7d567ac17/README.md?plain=1#L113).

Reply:
I was following the these steps from the Indexer docs and it was presented like this: 
https://thegraph.com/docs/indexing
![Screenshot from 2021-11-12 13-06-22](https://user-images.githubusercontent.com/36774944/141457524-85d4d7aa-95b5-45dd-8263-18643321f5e2.png)

But I think you're right, just I want an approve on this and will add it.

Reply:
Thank you so much for linking the same issue in the docs, I can fix it there ❤️

Reply:
You're welcome :) 
I will edit it now to be `NETWORK_NAME:[CAPABILITIES]:URL`